matrix/matrix_equalization.py

```diff
@@ -0,0 +1,47 @@
+import sys
+
+
+def array_equalization(vector: list[int], k: int) -> int:
```

Comment:
Self-documenting variable names, please.
```suggestion
def array_equalization(vector: list[int], step_size: int) -> int:
```